tests/unittests/CMakeLists.txt

```diff
@@ -335,7 +335,12 @@ target_link_libraries(PartitionerTest
                         Partitioner
                         gtest
                         TestMain)
-add_glow_test(PartitionerTest ${GLOW_BINARY_DIR}/tests/PartitionerTest --gtest_output=xml:PartitionerTest.xml)
+
+# There are multiple test cases in PartitionerTest that work only with CPU
```

Comment:
@beicy Is it really necessary to for the PartitionerTest to use the CPU backend?

Reply:
There are heterogeneous Partition test. So we use "CPU" and "Interpreter" for different type of backends.

Reply:
@beicy Could it be two different Mock backends, both Interpreter based, but with different sets of supported operations? Or do they need to be based on different backends?